### PR TITLE
Independent recurring

### PR DESCRIPTION
**Pull request by TalPat**
The portal for a recurring subscription is now an independent service.

### CODE REVIEW

File: sites/recurring/next.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const withCSS = require('@zeit/next-css')
 
 module.exports = withCSS({
     env: {
-        OAUTH_CLIENT_ID: 'flix.rafiki.shop',
+        OAUTH_CLIENT_ID: 'rafiki.shop',
```
Comment (matdehaast) [inline sites/recurring/next.config.js]: Both these values need to be configurable from environment variables

File: sites/recurring/pages/api/fetchaccess.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { NextApiRequest, NextApiResponse } from "next"
+const { Issuer } = require("openid-client")
+
+const OAUTH_CLIENT_ID = "rafiki.shop"
```
Comment (matdehaast) [inline sites/recurring/pages/api/fetchaccess.ts]: Don't hardcode, must come from next config https://nextjs.org/docs#runtime-configuration

File: sites/recurring/pages/index.tsx [hunk 3]
```diff
@@ -1,21 +1,294 @@
...
+	)
+}
+
+type Props = {
+	id: string
+}
+
+const Page: NextPage<Props> = ({ id }) => {
+
+	const OAUTH_CLIENT_ID = process.env.OAUTH_CLIENT_ID
```
Comment (matdehaast) [inline sites/recurring/pages/index.tsx]: https://nextjs.org/docs#runtime-configuration

Comment (TalPat) [reply sites/recurring/pages/index.tsx]: Changes have been made

**Pull request merged**